File: Change two-dimensional array.py
def matrix(array): 
    for i in range(0,len(array)):
        for j in range(0,len(array)):
            if i==j:
                tmp = array[i][j]
                if tmp<0:
                    array[i][j]=0
                else:
                    array[i][j]=1
    return array

File: Change two-dimensional array.js
function matrix(array) {
  for(let i=0;i<array.length;i++){
    for(let j=0;j<array.length;j++){
      if(i==j){
        if(array[i][j]<0){
          array[i][j]=0
        }else{
          array[i][j]=1
        }
      }
    }
  }
  return array
} 
